clang/test/dpct/cmake_migation/case_012/expected.txt

```diff
@@ -0,0 +1,7 @@
+add_library(culib)
+add_executable(cuexe)
+target_link_libraries(culib PUBLIC -L/lib/intel64 -lmkl_sycl -lmkl_intel_ilp64 -lmkl_tbb_thread -lmkl_core)
```

Comment:
How about introducing a helper function `sycl_add_mkl_to_target` to migrate `cuda_add_cufft_to_target`?
Because we need link different libraries on Linux and Windows.
Linux: `-lmkl_intel_ilp64 -lmkl_sequential -lmkl_core -lOpenCL -lmkl_sycl -lpthread -ldl`
Windows: `mkl_sycl_dll.lib mkl_intel_ilp64_dll.lib mkl_sequential_dll.lib mkl_core_dll.lib OpenCL.lib`

Reply:
@zhiweij1 good advice
@TejaX-Alaghari for sycl_add_mkl_to_target, you can create a file "SYCLomatic/clang/tools/dpct/cmake/sycl_helper.cmake" and add sycl_add_mkl_to_target to this file.
Then install the cmake helper file sycl_helper.cmake to "[dpct installation directory]/cmake/sycl_helper.cmake" of installation directory.

Reply:
Sure @tomflinda and @zhiweij1 
I'll attempt to do the same and get back.

Reply:
Can you please suggest whether its possible to include the helper file only once in the beginning of the file.
Maybe immediately after "project"?

Reply:
@TejaX-Alaghari the heper function has been merged into the code repo, you need update your PR to latest code repo.